src/bindings.js

```diff
@@ -266,31 +266,30 @@ export class ComponentBinding extends Binding {
   // Initializes a component binding for the specified view. The raw component
   // element is passed in along with the component type. Attributes and scope
   // inflections are determined based on the components defined attributes.
-  constructor(view, el, type) {
-    this.view = view
-    this.el = el
-    this.type = type
-    this.component = this.view.components[this.type]
-    this.static = {}
-    this.observers = {}
-    this.upstreamObservers = {}
-
-    let bindingRegExp = view.bindingRegExp()
-
-    if (this.el.attributes) {
-      this.el.attributes.forEach(attribute => {
-        if (!bindingRegExp.test(attribute.name)) {
-          let propertyName = this.camelCase(attribute.name)
-          let stat = this.component.static
-
-          if (stat && stat.indexOf(propertyName) > -1) {
-            this.static[propertyName] = attribute.value
-          } else {
-            this.observers[propertyName] = attribute.value
-          }
-        }
-      })
-    }
+    constructor(view, el, type) {
+		super(view, el, type);
+
+		this.component = this.view.components[this.type]
+		this.static = {}
+		this.observers = {}
+		this.upstreamObservers = {}
+
+		let bindingRegExp = view.bindingRegExp()
+
+		if (this.el.attributes) {
+		  this.el.attributes.forEach(attribute => {
+			if (!bindingRegExp.test(attribute.name)) {
+			  let propertyName = this.camelCase(attribute.name)
+			  let stat = this.component.static
+
+			  if (stat && stat.indexOf(propertyName) > -1) {
+				this.static[propertyName] = attribute.value
+			  } else {
+				this.observers[propertyName] = attribute.value
+			  }
+			}
+		  })
+		}
```

Comment:
Hey, sorry for the long delay, I have more time now and want to get Rivets up to date and in active development again. I want to merge your PR but could you fix the indentation here first? Thanks so much!